modules/load_balancer/variables.tf

```diff
@@ -85,3 +84,8 @@ variable "lb_tls_policy" {
   type        = string
   default     = "ELBSecurityPolicy-TLS13-1-2-2021-06"
 }
+
+variable "lb_allowed_cidrs" {
```

Comment:
I don't see this being used

Reply:
lb_allowed_cidrs              = var.allowed_inbound_cidrs_lb in the main.tf file